pf/internal/check/check_test.go

```diff
@@ -90,14 +127,26 @@ func test(t *testing.T, info tfbridge.ProviderInfo) (string, error) {
 }
 
 type (
-	testProvider struct{ provider.Provider }
-	testResource struct{ resource.Resource }
+	testProvider struct {
+		provider.Provider
+
+		missingID, sensativeID bool
```

Comment:
```suggestion
		missingID, sensitiveID bool
```

Reply:
> I don't generally block on spelling

Please do let me know if you spot a spelling mistake. I'll fix it before merging!